Sources/NIOWebSocket/WebSocketFrame.swift

```diff
@@ -0,0 +1,269 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+//
+//  WebSocketFrame.swift
+//  NIOWebSocket
+//
+//  Created by Cory Benfield on 16/02/2018.
+//
+
+import NIO
+
+private extension UInt8 {
+    func bitSetWithMask(_ mask: UInt8) -> Bool {
```

Comment:
that sounds like it would set a bit. Maybe should be named `isAnyBitSetInMask(_ mask: UInt8)`?